media/wav.ksy

```diff
@@ -0,0 +1,272 @@
+meta:
+  id: wav
+  file-extension: wav
+  title: Microsoft WAVE audio file
+  endian: le
+  license: BSD-3-Clause-Attribution
+  encoding: ASCII
```

Comment:
True that, but I know something Microsoft doesn't -- Microsoft stole this format from Electronic Arts, and EA designed the format at a time when it was ASCII or nothing.  ANSI hadn't been invented yet.  Ergo, ASCII.